### DISCUSSION TITLE
2021 07 26 회의록

### DISCUSSION THREAD

**Discussion by suhyeokkim** [KG-Crash/crash#50] [Announcements]
참석자 : 채승현, 김수혁

## 방향

지금 A*, 충돌, 맵을 일감을 나눴는데 쥰내 진행 안되고 앞으로 안될거 같아서, 채승현이 한꺼번에 하고 나머지 사람들은 게임 다른 구현을 맡는게 좋을 것 같다는 채승현의 의견이 있었다. 그리고 김수혁도 맞다고 생각했다. 

그래서 김수혁에게 나머지 일감을 전부 달아놓았다. 성주찡이 하고싶은거 있으면, 가져가고 없으면 우리가 임의로 배정해줄 예정임.

## 만든 일감 및 담당자

@boyism80 : #42 #43
@suhyeokkim : #44 #45 #46 #48 (여기서 성주찡이 하고싶은거 선택)
@Joogba : #47

**Comment by Joogba** [comment]
#47 먼저 해보겠습니다
